docs/release/release_template.rst

```diff
@@ -1,5 +1,7 @@
-Announcement: GraSPy 0.0.X
-==========================
+:orphan:
```

Comment:
can this go away now?

Reply:
No, because the template isn't actually in the docs.

Reply:
thx